crates/rolldown/tests/esbuild/default/re_export_default_external_common_js/artifacts.snap

```diff
@@ -0,0 +1,31 @@
+---
+source: crates/rolldown/tests/common/case.rs
+expression: content
+input_file: crates/rolldown/tests/esbuild/default/re_export_default_external_common_js
+---
+# warnings
+
+## UNRESOLVED_IMPORT
+
+```text
+[UNRESOLVED_IMPORT] Warning: "foo" is imported by "entry.js", but could not be resolved – treating it as an external dependency.
+
+```
+## UNRESOLVED_IMPORT
+
+```text
+[UNRESOLVED_IMPORT] Warning: "bar" is imported by "bar.js", but could not be resolved – treating it as an external dependency.
+
+```
+# Assets
+
+## entry_js.cjs
+
+```js
+const { __toESM } = require("./$runtime$.cjs");
+const { default: foo } = __toESM(require("foo"));
+const { default: bar } = __toESM(require("bar"));
+
+exports.bar = bar;
+exports.foo = foo;
+```
```

Comment:
Please recover this test case, 
the esbuild snapshot is
```js
================================================================================
TestReExportDefaultExternalCommonJS
---------- /out.js ----------
// entry.js
var entry_exports = {};
__export(entry_exports, {
  bar: () => import_bar.default,
  foo: () => import_foo.default
});
module.exports = __toCommonJS(entry_exports);
var import_foo = __toESM(require("foo"));

// bar.js
var import_bar = __toESM(require("bar"));

```

Reply:
could I ask, "recover this test case" means not to enable this case until snapshot be same with esbuild's?

Reply:
No need to revert. I will take a note on this case and fix it later. It's fine we merge them first. But still could you solve the `UNRESOLVED` warning.

Reply:
Got it, I solved them.